src/docs/build_files.md

```diff
@@ -104,6 +105,28 @@ You can also recursively glob over files in all subdirectories of the BUILD file
 However this is discouraged as it tends to lead to coarse-grained dependencies, and Pants's
 advantages come into play when you have many fine-grained dependencies.
 
+### tags
+
+Tags are a set of strings used to describe or categorize targets. They can be consumed by subsystems during a build to allow for features such as filtering task targets (ex. skip linting targets with a particular tag) or focused testing (ex. running only unit tests by excluding targets with a `integration` tag).
+
+Tags can be configured for targets in three ways:
+- Directly in the `BUILD` file (as above)
```

Comment:
I think the example target declaration is far enough away that this could use a little more verbiage pointing at it. Maybe 
```suggestion
- Directly in the `BUILD` file (`tags={'common'}` in the example above)
```